setuptools_scm/version.py

```diff
@@ -145,6 +148,31 @@ def guess_next_dev_version(version):
         return guess_next_version(version.tag, version.distance)
 
 
+def guess_next_simple_semver(version, distance, retain, increment=True):
+    parts = str(version).split('.', retain)
+    del parts[retain:]
+    while len(parts) < retain:
```

Comment:
These next two lines are padding the list of version numbers with zeros. I'd either document that with a comment and whitespace or extract a function whose docstring described the intention.